content/2004/eclipsecon/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "EclipseCon 2004"
+date: 2018-04-05T16:09:45-04:00
```

Comment:
We need to use the start date of the event here for our list of events to work. For example, /2004

Reply:
Ok I updated 2004, 2005, 2006 and france2015.